client/src/pages/Scheduler/Scheduler.tsx

```diff
@@ -99,12 +105,23 @@ export default function Scheduler(): JSX.Element {
             userTimeZone={userTimeZone}
             timeZone={timeZone}
             today={today}
-            duration={duration}
-            getDateInfo={getDateInfo}
+            duration={Number(duration)}
+            checkConfirmation={checkConfirmation}
             checkDisableTime={checkDisableTime}
           />
         </Box>
       </Box>
+      {confirmTrigger ? (
+        <Confirmation
+          username={username}
+          duration={Number(duration)}
+          timeZone={timeZone}
+          time={moment.tz(dateSelISO, timeZone)}
+          cancelConfirmation={cancelConfirmation}
+        />
+      ) : (
+        <Box />
+      )}
```

Comment:
```suggestion
      {confirmTrigger && (
        <Confirmation
          username={username}
          duration={Number(duration)}
          timeZone={timeZone}
          time={moment.tz(dateSelISO, timeZone)}
          cancelConfirmation={cancelConfirmation}
        />
      )}
```

Reply:
I forgot I am able to do this in react. Thank you!